### DISCUSSION TITLE
SegmentationFault with spectrum2_manager

### DISCUSSION THREAD

**Discussion by nicolas-it** [SpectrumIM/spectrum2#426] [Q&A]
Hi,
after manually compiling spectrum2 I get "segmentation fault" errors every time I'm using spectrum2.
This is a part of my syslog:
```

[2458769.337134] spectrum2[11492]: segfault at 7f58bbe76b20 ip 00007f58c57ffa53 sp 00007fff45cb4820 error 4 in libboost_signals.so.1.62.0[7f58c57f0000+17000]
[2458777.146518] spectrum2_manag[20609]: segfault at ff0000 ip 00007ffb76495e2d sp 00007ffe9912f7a8 error 6 in libc-2.24.so[7ffb7636d000+195000]
```

My environment is debian 9.
Am I hitting a wrong dependency of libboost?

**Comment by vitalyster** [comment]
http://spectrum.im/documentation/management/getting_backtrace.html

**Comment by nicolas-it** [comment]
Hi, I'm sorry for not providing a proper backtrace.
Unfortunatly I was not able to successfully build spectrum2 under debian stretch without dependency errors of libssl.
There seems to be a problem with libssl1.0 and libssl1.1
If anybody knows how to handle that, I can rebuild it and provide a backtrace of the error.

**Comment by vitalyster** [comment]
There are ready to use packages for Debian 9 including debug symbols, what is the problem? Moreover, you also need boost debug symbols
